File: 프로그래머스/unrated/181875. 배열에서 문자열 대소문자 변환하기/배열에서 문자열 대소문자 변환하기.cs
using System;
using System.Linq;

public class Solution {
    public string[] solution(string[] strArr) 
    {
        return strArr.Select((c, index) => 
                         {
                             if(index % 2 == 0) return c.ToLower();
                             else return c.ToUpper();
                         }).ToArray();
    }
}

File: 프로그래머스/0/181875. 배열에서 문자열 대소문자 변환하기/배열에서 문자열 대소문자 변환하기.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<string> solution(vector<string> strArr) {
    
    for (int i = 0; i < strArr.size(); i += 2)
    {
        std::transform(strArr[i].begin(), strArr[i].end(), strArr[i].begin(), ::tolower);
    }

    for (int i = 1; i < strArr.size(); i += 2)
    {
        std::transform(strArr[i].begin(), strArr[i].end(), strArr[i].begin(), ::toupper);
    }
    
    return strArr;
}